requirements.txt

```diff
@@ -1,16 +1,16 @@
 # Requirements for the Call Tracking sample Django project
 
 # Django (1.8 is a long-term support release)
-Django==1.8
+Django==3.2.0
 
 # Third-party app dependencies
-django-bootstrap3==6.1.0
-django-forms-bootstrap==3.0.1
+django-bootstrap3==21.1.0
+django-forms-bootstrap==3.1.0
 django-phonenumber-field==0.7.2
 phonenumbers==7.3.0
 psycopg2==2.8.6
 twilio==6.9.0
-whitenoise==2.0.2
+whitenoise==5.3.0
```

Comment:
Is whitenoise still used in the project? I see that some references are deleted.

Reply:
I was thinking that it was only for static files management and I removed it, but it's for serving them from what I was checking, I fixed the implementation and configurations and reintroduced it.